Lesson27/ezgi/create/create-post.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Create post</title>
+		<link rel="stylesheet" type="text/css" href="../styles.css" />
+	</head>
+	<body>
+		<div class="container">
+			<h1>Create post</h1>
+      
+		<form action="" id="create-post">
+			<label for="title">Title :</label>
+			<textarea id="title"></textarea>
+
+			<label for="body">Paragraph</label>
+			<textarea id="body"></textarea>
+			<button id="Create-post-button" type="submit">Create Post</button>
+		</form>
+		</div>	
+		<div id="successError"></div>
```

Comment:
In this file, add an extra button (`<a>` element) to the page that would redirect the user back to the main page.

Reply:
I've added the path (Lesson27/ezgi/index.html) however this's trying to open the another page (Lesson27/ezgi/create/Lesson27/ezgi/index.html) . I think that because this file is in the create folder. But I couldnt find solution